dashboard/emails.py

```diff
@@ -74,3 +74,22 @@ def account_rejection_email(user):
             "and unfortunately it has been rejected. For any questions you "\
             "may have please contact us at {}".format(DASH_SUPPORT)
     send_email(subject, body, recipient=user.email)
+
```

Comment:
I missed this previously but on line 11, `async` is a reserved keyword in Python 3.7. Should the function name be changed?

Reply:
Good catch! Currently I don't think the dashboard is at all python3 compatible but I'll change it now so I don't shoot myself in the foot later.